portfolio

quavo19 opened the pull request:
### in this project 

- I followed the CSS and HTML best practices 
- I made a mobile version of a home page for my portfolio website
- I set up all linter files 
- I developed a README.md file and clearly documented my project

Reem-lab commented on index.html:
```diff
@@ -7,7 +7,7 @@
     <title>Mobile First</title>
     <link rel="stylesheet" href="style.css">
-</head>
+</head> 
```
- [ ] Great job working on the design but I think in the bottom of the page you need to fix it by adding some margin/padding to it so kindly stick with the figma as much as possible as required 🙏🏻😊🔥 and remeber to make the border as curved in the side😊🔥🔥

- Your design
![image](https://user-images.githubusercontent.com/58553711/198170132-6a28c70e-0b6d-432e-b09f-9c8278c3a29b.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/198170187-18e17a29-6729-470c-99e9-34c6545ed6a2.png)

Reem-lab commented on README.md:
```diff
@@ -1,21 +1,37 @@
 # Mobile-First
 Project Name
-Portfolio 
+Simple mobile form of mu portfolio
```
- [x] Great job but I think you forget to customzie Microverse README [template](https://github.com/microverseinc/readme-template) into your information and project's information and remembre it is very important because it showcase your project and what is about? and some steps about how to clone the project if someone want to clone it and it shows who contributed with this project? so that's very important in the future when searching for a job so kindly add it as required 💪🏻☺💯
![image](https://user-images.githubusercontent.com/58553711/197897153-848edbfe-ee71-4de8-b889-ed157f87a587.png)

- [ OPTIONAL ] I recommend add more description about what this project about so you describe that this a portfolio so you can show your projects and skills and contact form 🔥😊🙏🏻

Reem-lab reviewed (changes requested):
# Hi @quavo19, 👋🏻👋🏻 

Good job so far! 🔥🚀💪🏻💯💯
There are some issues that you still need to work on to go to the next project but you are almost there!
<img src="https://media3.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif"/>


## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the Functionality ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._
- [ ] I suggest wrapping your logo with `a` tag so in the future you can press on it and naviagte to the first page and the user can be able to navigate easily as well  and This is for three reasons: convention, semantics, and user experience(UX)💯💪🏻🚀

- [ ] For the heading I think it would better if you keep it `h1` tag to make your HTML semantic but remember in the future you should also have one `h1` tag  per page to follow the SEO rules 🚀💪🏻
and add some font wight to make identical with the Figma template 💪🏻🚀💯

- Your design
![image](https://user-images.githubusercontent.com/58553711/198170940-44e2e1a6-4409-4a9c-87a5-ad41f4ed66ee.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/198171007-32de6d89-0955-4acc-8e0e-36b028749b40.png)

- [ ] I suggest adding header `tag` and inside it put your logo and you humburger button wrapped with  a `nav` tag that's very important for semantics reason so kindly update it 💯🚀💪🏻

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

quavo19 commented:
### in this section I;

- fixed the height of box-shadow 
- fixed linter errors 
- followed CSS and HTML best practices 
- added a README.md text

Owusu-Desmond commented on README.md:
```diff
@@ -1,21 +1,35 @@
 # Mobile-First
 Project Name
-Portfolio 
+Simple mobile form of mu portfolio
```
- [ ] Please make sure you fix this typo `mu` which I guess to be `my` :smiley:

Owusu-Desmond commented on README.md:
```diff
@@ -1,21 +1,35 @@
+Inspiration
+etc
+📝 License
+This project is MIT licensed.
```
- [ ] Please you should use hashtags for the titles in your `readme.md` file to make it more readable instead of using raw text. :pray:. As requested by the previous code reviewer you should follow the README [template](https://github.com/microverseinc/readme-template) which contains all sections.

  - HIT: You can copy all the sections in the template and modify it with your project details. :+1:

Owusu-Desmond commented on index.html:
```diff
@@ -3,34 +3,38 @@
-     <div class="header">
-        <div class="nav-bar">
-    
-                    <div class="logo">
-                        <svg width="75" height="18" viewBox="0 0 75 18" fill="none" xmlns="http://www.w3.org/2000/svg">
-                            <path d="M14.7548 0.364V13H11.6768V5.098L8.94081 13H6.27681L3.52281 5.08V13H0.444813V0.364H4.20681L7.64481 9.166L11.0108 0.364H14.7548ZM19.2024 2.956L21.6144 9.112L23.8644 2.956H27.2664L21.0024 17.788H17.6184L19.9764 12.622L15.7644 2.956H19.2024ZM34.9701 10.642H39.0741V13H31.8921V0.274H34.9701V10.642ZM45.1858 2.83C46.1818 2.83 47.0698 3.04 47.8498 3.46C48.6418 3.868 49 ...
-                            </svg>
```
- [x] As requested by the previous reviewer, you should wrap your logo with `a` so someone can easily navigate your website by clicking on the logo to get back to the home page. :100:

quavo19 replied:
@Owusu-Desmond  please the logo is wrapped with an a tag already

Owusu-Desmond reviewed (changes requested):
Hi @quavo19  :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.

<img src="https://media4.giphy.com/media/xUOxfh6ZM75efM3Bqo/giphy-downsized-medium.gif"/>

 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

quavo19 commented on index.html:
```diff
@@ -3,34 +3,38 @@
-          
-                  </div>
+     
+        <header class="nav-container">
```
I wrapped the logo. here you are reading from a deleted line

quavo19 commented on index.html:
```diff
@@ -3,34 +3,38 @@
+                                <path d="M14.7548 0.364V13H11.6768V5.098L8.94081 13H6.27681L3.52281 5.08V13H0.444813V0.364H4.20681L7.64481 9.166L11.0108 0.364H14.7548ZM19.2024 2.956L21.6144 9.112L23.8644 2.956H27.2664L21.0024 17.788H17.6184L19.9764 12.622L15.7644 2.956H19.2024ZM34.9701 10.642H39.0741V13H31.8921V0.274H34.9701V10.642ZM45.1858 2.83C46.1818 2.83 47.0698 3.04 47.8498 3.46C48.6418 3.86 ...
+                                </svg>       
+                       </div>
+                     </a>
```
the tag ended here 
so you are reading from the lines I deleted

anagudelogu reviewed (approved):
Hi @quavo19, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🍾 🎆  
Congratulations! 🎉 

<img src="https://media2.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif"/>

## Highlights
✅  No Linters error 
✅  Descriptive pull request
✅  Good commit messages
✅ Great responsive design! 🥇 


## Optional suggestions
- [ ] **[OPTIONAL]:** You could stick as much as possible to the [Microverse's README template](https://github.com/microverseinc/readme-template). 📜 This includes the markdown format (which is the one given by hashtags `#`, for example. It would be good to get a little familiar with `.md` syntax for your future project's documentation. Remember that having professional documentation makes your project more attractive to others! 🍾 🛠️ 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
